platform/solr/solr-banana/src/main/resources/banana-int/Banana-Default-Dashboard.json

```diff
@@ -0,0 +1,550 @@
+{
+  "title": "Metadata Default Dashboard",
```

Comment:
Hmm, I figured Metacard would work as an umbrella term to encompass all possible query data that gets ingested.